### PR TITLE
Use `prop-types` instead of `React.PropTypes`

### PR DESCRIPTION
**Pull request by jochenberger**
`React.PropTypes` is deprecated as of React 15.5.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -53,7 +53,8 @@
     "react": "^0.14.0 || ^15.0.0"
   },
   "dependencies": {
-    "attr-accept": "^1.0.3"
+    "attr-accept": "^1.0.3",
+    "prop-types": "^15.5.7"
```
Comment (okonet) [inline package.json]: Should this go to `devDependencies` and `peerDependencies` since it's React's dependency and not ours?

**Comment by okonet** [comment]
Will it work with older React versions or this must be a breaking change?

File: package.json (hunk 1 above)
Comment (jochenberger) [inline package.json]: See https://www.npmjs.com/package/prop-types#how-to-depend-on-this-package

**Comment by jochenberger** [comment]
See https://www.npmjs.com/package/prop-types#compatibility

**Comment by andrevenancio** [comment]
I just pulled from `npm` version 3.12.3 and on the `src` folder there's still React.PropTypes instead of PropTypes...

**Comment by okonet** [comment]
Probably because of https://github.com/okonet/react-dropzone/blob/master/src/index.js#L378?

@jochenberger mind creating a new PR that removes it as well?

**Comment by andrevenancio** [comment]
Not sure if that's the reason. shouldn't I have the exact same code I see here on the master branch on git when I pull that tgz file from npm? I've pulled it, and the src/index.js still has `React.PropTypes`

**Comment by jochenberger** [comment]
The relevant change is not in 3.12.3 AFAICT.

**Comment by okonet** [comment]
I think `chore` commit type doesn't trigger a release. I'll create an empty commit with `fix` to trigger a build

**Comment by okonet** [comment]
Or rebase master 😱

**Pull request merged**